serverless-heaven/serverless-webpack
# Extends the --no-build option to serverless offline start

## What did you implement:
Extends the --no-build option to `serverless offline start` and `serverless offline`, allowing the ability to use previously compiled files from either a previous `serverless offline start` (when keeping output files) or from a direct `serverless webpack`

Closes #415 



## How did you implement it:

skips wpwatch and sets skipCompile to true when --no-build is set in the CLI.


## How can we verify it:

added 2 tests to verify `serverless offline start --no-build` doesn't compile output again.


## Todos:


***Is this ready for review?:*** NO
***Is it a breaking change?:*** NO
